src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories.Tests/DlcsSynchronisation/ProcessingBehaviourTests.cs

```diff
@@ -18,10 +25,10 @@ private PhysicalFile MakePhysicalFile()
     [Fact]
     public void Image_Has_IIIF_and_thumbs_Delivery_Channel()
     {
-        var pf = MakePhysicalFile();
-        pf.MimeType = "image/jpg";
+        var storedFile = MakeStoredFile();
+        storedFile.MimeType = "image/jpg";
 
-        var processing = pf.ProcessingBehaviour;
+        var processing = storedFile.ProcessingBehaviour;
 
         processing.DeliveryChannels.Should().OnlyContain(s => s == "iiif-img" || s == "thumbs");
```

Comment:
Should this tests be updated to remove thumbs? `DeliveryChannels` is only "iiif-img" here but it passes.